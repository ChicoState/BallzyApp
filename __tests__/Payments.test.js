'use strict';

import 'react-native';
import React from 'react';
//import React from 'react';
import Pay from '../src/components/payments/Payments';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <Pay />
  ).toJSON();
  expect(tree).toMatchSnapshot();

});
