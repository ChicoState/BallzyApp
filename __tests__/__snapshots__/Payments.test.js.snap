exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#dcebfc",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 40,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }>
      Payments
    </Text>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#4392f1",
          "borderRadius": 5,
          "height": 45,
          "justifyContent": "center",
          "margin": 15,
          "opacity": 1,
          "width": 150,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "color": "#FFFFFF",
            "fontSize": 15,
            "padding": 5,
            "textAlign": "center",
          }
        }>
        Pay with Stored Card
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#4392f1",
          "borderRadius": 5,
          "height": 45,
          "justifyContent": "center",
          "margin": 15,
          "opacity": 1,
          "width": 150,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "alignSelf": "center",
            "color": "#FFFFFF",
            "fontSize": 15,
            "padding": 5,
            "textAlign": "center",
          }
        }>
        Add New Card
      </Text>
    </View>
  </View>
</View>
`;
